lib/rules/web/ocm_console/dialogs_and_sidebar.xyaml

```diff
@@ -177,27 +177,28 @@ close_error_message_dialog:
             type: submit
         op: click
 
-edit_compute_node_in_dialog:
-  action: clear_original_compute_node_in_dialog
-  action: input_compute_node_in_dialog
+choose_compute_node_in_dialog:
```

Comment:
I think choose_compute_node_in_dialog is an action should  happened in a dialog. The actions that save and check_dialog_missing shouldn't be included